engine/src/org/pentaho/di/trans/steps/mappinginput/MappingInput.java

```diff
@@ -165,21 +176,6 @@ public boolean processRow( StepMetaInterface smi, StepDataInterface sdi ) throws
     return true;
   }
 
-  private void replaceRenameIfNecessary( RowMetaInterface rowMeta, MappingValueRename valueRename )
```

Comment:
@deinspanjer, 
It is a revert commit. I created this function in previous PR (that doesn't fix the problem) - https://github.com/pentaho/pentaho-kettle/pull/1027.